spec/rubocop/cop/rspec/receive_messages_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ReceiveMessages, :config do
```

Comment:
For spec completeness, can we add:
* allow with block syntax
`allow(Service).to receive(:foo) { baz }`
* allow with `with`
`allow(Service).to receive(:foo).with(:bar).and_return(:baz)`
* different return value types: array, hash and especially heredoc. What if we have two messages returning heredocs?